sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/_configuration/_utils.py

```diff
@@ -11,9 +11,36 @@
     _ONE_SETTINGS_CHANGE_VERSION_KEY,
 )
 
+
 logger = logging.getLogger(__name__)
 
 
+class _ConfigurationProfile:
```

Comment:
Config profile identifies info about the sdk that is used for when determining which sdks should have features turned on.